modules/extension/xsd/xsd-filter/src/main/java/org/geotools/filter/v1_0/OGCUtils.java

```diff
@@ -99,7 +99,7 @@ static Object property(Expression e1, Expression e2, QName name) {
      * @param node The parse tree.
      * @return A two element array of expressions for a BinarySpatialOp type.
      */
-    static Expression[] spatial(Node node, FilterFactory2 ff, GeometryFactory gf) {
+    public static Expression[] spatial(Node node, FilterFactory2 ff, GeometryFactory gf) {
```

Comment:
FES bindings seem to build off the OGC ones, but I'm still concerned about the solution if I need to make this public. Thoughts?

Reply:
The utility class indeed did not seem to be meant for public consumption.
How about rolling a method to parse units in the base class, and have the FES version override it to pick the uom from the different attribute?